pages/index.tsx

```diff
@@ -21,15 +32,11 @@ export default function Home() {
       <section className={styles.container}>
         <main className={styles.MainArea}>
           <HeroBanner />
-          {/* In the config we have a list of sections, this is how we iterate through them */}
-          {sections.map((section, i) => (
-            <section key={i}>
-              <Section sectionConfig={section} index={i} />
-            </section>
-          ))}
+          {renderSections}
         </main>
         <Footer />
       </section>
+      <HackathonModal onRequestClose={() => {}} />
```

Comment:
Can we remove the `onRequestClose` prop. My logic here is this component is already setup todo one thing and one thing only lets not make it configurable if we dont need to.